build/changelog/build-changelog.php

```diff
@@ -4,8 +4,14 @@
 
 use Aws\Build\Changelog\ChangelogBuilder;
 
-$changelogBuilder = new ChangelogBuilder("", "", true);
+$params = [
+    'verbose_flag' => true
+];
+
+$changelogBuilder = new ChangelogBuilder($params);
```

Comment:
The default behavior is to run it from the base directory. The option for outputDir was added for unit test to test the methods and that Changelog is written at the correct place. 
The script will be called from base directory and will not need any variables to run normally.